bundle-workflow/README.md

```diff
@@ -242,14 +242,21 @@ The following options are available.
 
 ### Component Onboarding
 
-1. Update a [manifest](/manifests) for a particular release to include your plugin.  For example to be included in the 1.1.0 release, you would update [opensearch-1.1.0.yml](/manifests/1.1.0/opensearch-1.1.0.yml). We require your plugin name, repository url, and git ref that should be used. For unreleased versions this should be a branch in your repository.  Once a release is cut, these refs will be updated to build from a tag or specific commit hash.
+With many components included in the distribution each component bears responsibility to keep the release process streamlined and react to integration issues.
+
+1. Components repositories have a label matching the manifest versions so incoming issues can be appropriately labeled, e.g. Add `v1.1.0` to https://github.com/opensearch-project/opensearch-build/labels.
```

Comment:
Lowercase add. I would actually change this to “ensure” because we use plug-in meta to create labels across all our component repos. You can link that but it’s probably overkill. Either way use a component repo as an example and link that label example to issues labeled as such in  that repo.